tests/client_auth.rs

```diff
@@ -64,5 +64,17 @@ fn cert_with_both_ekus_accepted_for_client_auth() {
 fn cert_with_serverauth_eku_rejected_for_client_auth() {
     let ee = include_bytes!("client_auth/cert_with_serverauth_eku_rejected_for_client_auth.ee.der");
     let ca = include_bytes!("client_auth/cert_with_serverauth_eku_rejected_for_client_auth.ca.der");
-    assert_eq!(check_cert(ee, ca), Err(webpki::Error::RequiredEkuNotFound));
+    let err = check_cert(ee, ca).unwrap_err();
+    assert_eq!(
+        err,
+        webpki::Error::RequiredEkuNotFoundContext(RequiredEkuNotFoundContext {
+            required: KeyUsage::client_auth(),
+            present: vec![vec![43, 6, 1, 5, 5, 7, 3, 1]],
+        })
+    );
+
+    assert_eq!(
+        format!("{err}"),
+        "RequiredEkuNotFoundContext(RequiredEkuNotFoundContext { required: KeyUsage { inner: RequiredIfPresent(KeyPurposeId { oid_value: Input }) }, present: [[43, 6, 1, 5, 5, 7, 3, 1]] })"
+    )
```

Comment:
These changes need to be made in the Python [test generator code](https://github.com/rustls/webpki/blob/ab506142004db3f38957b3e3cf22244c7daf2936/tests/generate.py#L839-L892). Right now [CI is flagging the diff](https://github.com/rustls/webpki/actions/runs/14106892570/job/39515397207) that results from regenerating.